examples/github-appcards/README.md

```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
 
 ## Required Environment Variables
 
+To run the app, Vite loads an `.env` file with the [environment variables](https://vitejs.dev/guide/env-and-mode.html#env-files).
+Use the commented example below to create your `.env` file.
+
 ```.env
+# Generate an access token in GitHub, and enter the value here.
+# To generate the access token, go to https://github.com/settings/tokens
 VITE_GH_ACCESS_TOKEN=
+
+# In Supabase, you create a Postgres database, and you assign it a password.
+# Enter here the password of the database created in Supabase.
+# For more information, see:
+# https://supabase.com/docs/guides/database/managing-passwords
 VITE_SUPABASE_PASSWORD=
+
+# Enter here the URL of the Supabase database that the app uses for data persistence.
+# For more information, see:
+# https://supabase.com/docs/guides/database
 VITE_DATABASE_URL=
+
+# Enter here the API key of the Supabase project with the database that the app uses for data persistence.
+# For more information, see:
+# https://supabase.com/docs/guides/api#api-url-and-keys
 VITE_DATABASE_PUBLIC_KEY=
+
+# Enter here the base URL of the hosting service your app runs on.
+# If you're developing locally, it can be 'localhost'.
 VITE_BASE_URL=
-VITE_MIRO_API_TOKEN=
```

Comment:
Was this one removed on purpose?

Reply:
@janza yes, based on this comment in https://github.com/miroapp/app-examples/issues/80:

```
# Ignore this field, it's no longer necessary. We'll update the README accordingly.
VITE_MIRO_API_TOKEN=
```

We don't use this env var anywhere inside this repo.
Do you know where else I can check to make sure we really don't need it?

Reply:
Update:
`VITE_MIRO_API_TOKEN` is not used anywhere. It's safe to remove it from the `.env` file.